src/invidious/views/components/item.ecr

```diff
@@ -45,7 +45,7 @@
                 <p dir="auto"><%= HTML.escape(item.title) %></p>
             </a>
             <a href="/channel/<%= item.ucid %>">
-                <p dir="auto"><b><%= HTML.escape(item.author) %></b></p>
+                <p dir="auto"><b><%= HTML.escape(item.author) %><% if !item.is_a?(MixVideo) && !item.author_verified.nil? && item.author_verified %> <i class="icon ion ion-md-checkmark-circle"></i><% end %></b></p>
```

Comment:
In this part of the code, you're only dealing with a `MixVideo` (note the `when MixVideo` on line 35).
That means that you don't need to do any changes that line, as it seems that you want to exclude `MixVideo` kind of videos. They don't have an owner badge structure?